src/components/Pagination.vue

```diff
@@ -60,7 +60,7 @@ export default {
     isLastQuestion(){
       const objectID = this.$route.params.id;
       const pageNum = JSON.parse(objectID);
-      if(pageNum === 7) {
+      if(pageNum === 8) {
```

Comment:
Might be worth considering making this number dynamic. Using `.length` for the questions array would allow you to do this